charts/moco/CHANGELOG.md

```diff
@@ -5,6 +5,46 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Support MySQLCluster v1beta2 API
```

Comment:
Manual operations are required to support conversion webhooks in Helm Chart, which is documented in the changelog.